.github/workflows/cmake.yml

```diff
@@ -0,0 +1,66 @@
+name: CMake
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    # Skip building pull requests from the same repository
+    if: ${{ github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name != github.repository) }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      # TODO: replace extra-cmake-flags with CMake presets
+      matrix:
+        target:
```

Comment:
@c3rb3ru5d3d53c if you want to enable other platforms you can add an entry to the `target` list and the corresponding variables in `include`. Example I did yesterday here: https://github.com/li-script/lightning/blob/master/.github/workflows/cmake.yml#L13

Reply:
Perfect I'll look into that soon enough :)